docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -1,3 +1,154 @@
+After fine-tuning, the embeddings of positive pairs are expected to be pulled closer, while the embeddings for negative pairs are expected to be pushed away.
+
+
+## Preparing data
+Training and evaluation data are already prepared and pushed to Hubble following the [instructions](../2_step_by_step/2_4_create_training_data.md).
+You can either pull the data:
+```python
+from docarray import DocumentArray
+train_data = DocumentArray.pull('resnet-ttl-train-data')
+eval_data = DocumentArray.pull('resnet-ttl-eval-data')
+```
+Or specify given `DocumentArray` names (`resnet-ttl-train-data` and `resnet-ttl-eval-data`) directly to Finetuner.
+
+## Login to Finetuner
+As explained in the [Login to Jina ecosystem](../2_step_by_step/2_3_login_to_jina_ecosystem.md) section, first we need to login to Finetuner:
+```python
+import finetuner
+finetuner.login()
+```
+
+## Choosing the model
+Now let's see what backbone models we can use. You can see available models either in [the docs](../2_step_by_step/2_5_choose_back_bone.md) or by calling:
+```python
+finetuner.describe_models()
+```
+
+```bash
+                                                                  Finetuner backbones                                                                   
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃                                            model ┃           task ┃ output_dim ┃ architecture ┃                                          description ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│                                         resnet50 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                        resnet152 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b0 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b4 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                     openai/clip-vit-base-patch32 │  text-to-image │        768 │  transformer │ Pretrained on millions of text image pairs by OpenAI │
+│                                  bert-base-cased │   text-to-text │        768 │  transformer │       Pretrained on BookCorpus and English Wikipedia │
+│ sentence-transformers/msmarco-distilbert-base-v3 │   text-to-text │        768 │  transformer │           Pretrained on Bert, fine-tuned on MS Marco │
+└──────────────────────────────────────────────────┴────────────────┴────────────┴──────────────┴──────────────────────────────────────────────────────┘
+```
```

Comment:
not sure if it is needed, in every page, maybe we can just refer to the previous step-by-step secant. also explain a bit in this case, we're working on image to iamge search, so we can take a CNN model for fine-tuning